src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -792,6 +792,22 @@
     <value>Awaiting authentication completion from browser...</value>
     <comment>Text displayed after the user clicks the button to initiate the GitHub authentication flow in their browser.</comment>
   </data>
+  <data name="AISettings_Active" xml:space="preserve">
+    <value>Active</value>
+    <comment>Text displayed when the service provider is active.</comment>
+  </data>
+  <data name="AISettings_LoggedIn" xml:space="preserve">
+    <value>Logged in</value>
+    <comment>Text displayed when the user is logged in to the service provider.</comment>
+  </data>
+  <data name="AISettings_ActiveLoggedIn" xml:space="preserve">
+    <value>Active, Logged in</value>
+    <comment>Text displayed when the user is logged in to the service provider and that service provider is active.</comment>
+  </data>
+  <data name="AISettings_ProviderNotAllowed" xml:space="preserve">
+    <value>Managed by your administrator</value>

```

Comment:
```suggestion
    <value>Disabled by your administrator</value>
```

Doesn't this makes more sense. Because if it is not disabled we need no such a text and the user can use it or not.

Reply:
I support this